app/Http/Controllers/Backend/DashboardController.php

```diff
@@ -5,16 +5,85 @@
 use Illuminate\Http\Request;
 use App\Http\Requests;
 use App\Http\Controllers\Controller;
+use App\Models\Backend\Order;
+use DB;
 
 class DashboardController extends Controller
 {
     /**
      * Index admmin
      *
+     * @param string $condition this is of group by
+     *
+     * @return view index of admin
+     */
+    public function index($condition = 'DAYNAME')
```

Comment:
[rank B] we should use default parameter as NULL or empty only, don't set default value with other.